elastic/fleet-server
# deleting `secret_references` from computed agent policy

## What is the problem this PR solves?

Cleanup

## How does this PR solve the problem?

Remove `secret_references` from the computed agent policy being sent out to agents.

## How to test this PR locally

Add an integration with secrets, enroll an agent, check in agent diagnostics that the agent policy doesn't contain `secret_references`

## Design Checklist




## Checklist





## Related issues


Related to https://github.com/elastic/kibana/issues/154715
